cmd/cli/install_test.go

```diff
@@ -290,14 +290,6 @@ var _ = Describe("Running the install command", func() {
 			err := installCmd.run(config)
 			Expect(err.Error()).To(ContainSubstring("osm.vault.host is required"))
 		})
-
-		It("should error when token isn't set", func() {
```

Comment:
Could we replace this with a test that makes sure either `token` or the secret ref is set?